omegaconf/omegaconf.py

```diff
@@ -620,6 +623,32 @@ def to_container(
             structured_config_mode=structured_config_mode,
         )
 
+    @staticmethod
+    def to_object(
+        cfg: Any,
+        *,
+        resolve: bool = True,
+        enum_to_str: bool = False,
+    ) -> Union[Dict[DictKeyType, Any], List[Any], None, str, Any]:
+        """
+        Resursively converts an OmegaConf config to a primitive container (dict or list).
+        Any DictConfig objects backed by dataclasses or attrs classes are instantiated
+        as instances of those backing classes.
+
+        This is an alias for OmegaConf.to_container(..., resolve=True, structured_config_mode=SCMode.INSTANTIATE)
```

Comment:
Remove `resolve=True`?

Reply:
Remove from the docstring?
Or do you mean remove from the function signature?

Reply:
The motivation for resolve=True by default:
This enables the result of `to_object(cfg)` to be immediately used in the client's application logic, including e.g. calling methods belonging to the dataclasses that were defined by the client.

Reply:
I meant remove from the docstring, since my understanding is that `...` represents the arguments of `to_object()` (otherwise I would have expected `enum_to_str=False` to also be present in the docstring)

Reply:
Oh, that makes sense. Will do.

Reply:
Done in https://github.com/omry/omegaconf/pull/502/commits/a10fa5bdca987120e33a26a7e57fcd588aa8f2e8.

Reply:
Reverted in https://github.com/omry/omegaconf/pull/502/commits/fe5df1d6202ff18f92d3352be3c8d6e7bcd8ff80, since the `to_object` signature no longer takes a `resolve` argument.